docs/DeveloperGuide.md

```diff
@@ -10,7 +10,8 @@ title: Developer Guide
 
 ## **Acknowledgements**
 
-* {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+* Libraries used: [JavaFX](https://openjfx.io/), [Jackson](https://github.com/FasterXML/jackson), [JUnit5](https://github.com/junit-team/junit5), [OpenCSV](https://opencsv.sourceforge.net/)
```

Comment:
```suggestion
* Libraries used: [JavaFX](https://openjfx.io/)
* [Jackson](https://github.com/FasterXML/jackson)
* [JUnit5](https://github.com/junit-team/junit5)
* [OpenCSV](https://opencsv.sourceforge.net/)
```
We could perhaps split them up for tidiness

Reply:
Think it's alright to leave them as a single line! It looks better than it seems since they're each just one-word hyperlinks :D